### DISCUSSION TITLE
Random Top bar appears with split mode, and for the life of me, i cant get rid of it.

### DISCUSSION THREAD

**Discussion by pnelego** [i3/i3#6011] [Q&A] [answered]
Hi All, seemingly at random (probably me pressing a rouge keyboard shortcut),
this random bar appears and takes over my entire workspace. By default when it appears it seems to stack new windows. But when i switch to normal tiling mode, the bar doesn't go away, and windows loose their gaps. 

![Screenshot_2024-04-16_10-25-11](https://github.com/i3/i3/assets/11396340/875b0361-e7bb-4f20-8b29-11db17ad7e73)

I'm assuming I'm pressing some shortcut by accident. but honestly I don't know what I don't know here. Any help is appreciated :) 

Thanks!

**Comment by orestisfl** [comment, marked as answer]
This happens when you set the workspace itself on stacked (this screenshot) or tabbed mode. To avoid it, set individual windows to stacked instead.
